docs/script.rst

```diff
@@ -332,19 +328,19 @@ while the ``=~`` creates a Java ``Matcher`` object.
 Regular expression support is imported from Java. Java's regular expression language and API is documented in the
 `Pattern Java documentation <http://download.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html>`_.
 
-You may also be interested in this post `Groovy: Don't Fear the RegExp <http://naleid.com/blog/2008/05/19/dont-fear-the-regexp>`_.
+You may also be interested in this post: `Groovy: Don't Fear the RegExp <https://web.archive.org/web/20170621185113/http://www.naleid.com/blog/2008/05/19/dont-fear-the-regexp>`_.
```

Comment:
This page appears to be gone, but I found it on the Wayback Machine. Maybe there's a current link that could take its place? In any case, a link to a Wayback Machine page is probably better than a dead link.

Reply:
can't a better one, it's fine to use the one in the Wayback Machine